docs/releases/v0.3.0.md

```diff
@@ -0,0 +1,292 @@
+
+> [!IMPORTANT]  
+> Be sure to have enough space for the provisioning of PVCs during the migration.
+
+Edit the `Kafka` resource and add the `strimzi.io/kraft: disabled` annotation. Apply the change.
+
+Create the `KafkaNodePool` resource for the KRaft controllers, here is an example:
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaNodePool
+metadata:
+  name: controller
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  replicas: 3
+  roles:
+    - controller
+  storage:
+    type: jbod
+    volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+```
+
+Then change the annotation on the corresponding `Kafka` resource:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+  annotations:
+    strimzi.io/kraft: migration # updated from "disabled" to "migration" to trigger the migration process
+    strimzi.io/node-pools: enabled
+# ...
+```
+
+Apply the changes and wait for the operator to start the migration. Then monitor the cluster and wait that it reaches the `KRaftPostMigration` state. Then there are two choices:
+
+- Finalize the migration to KRaft;
+- Rollback to ZooKeeper.
+
+#### Finalize the migration to KRaft
+
+To finalize the migration the annotation `strimzi.io/kraft` needs to be changed to `enabled`, as follows:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+  annotations:
+    strimzi.io/kraft: enabled # updated from "migration" to "enabled" to finalize the migration process
+    strimzi.io/node-pools: enabled
+# ...
+```
+
+The operator will configure the KRaft controllers without the ZooKeeper connection details, with the migration flag disabled, and roll them. When both brokers and controllers are working in KRaft mode, the operator will delete all the resources related to ZooKeeper as well. At this point, the user can delete the .spec.zookeeper section from the Kafka custom resource because it is not needed anymore.
+
+#### Rollback to ZooKeeper
+
+If the migration process is still not finalized, with the KRaft controllers remaining connected to ZooKeeper, it is still possible to initiate a rollback. By applying the rollback value on the `strimzi.io/kraft` annotation, the operator reconfigures the brokers with the connection details to ZooKeeper again and rolls them. Here is an example:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+  annotations:
+    strimzi.io/kraft: rollback # updated from "migration" to "rollback" to revert the migration process
+    strimzi.io/node-pools: enabled
+# ...
+```
+Following the rollback, the cluster metadata state moves back to `KRaftDualWriting`, waiting for the user to finalize the process.
```

Comment:
````suggestion
```

Following the rollback, the cluster metadata state moves back to `KRaftDualWriting`, waiting for the user to finalize the process.
````